CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to the "Espressif IDF" extension will be documented in this file.
 
+## 0.4.0
+
+### System View Tracing Viewer, Search IDF Documentation, etc.
+
+- Add System View Tracing [support]() inside vscode, which will enable you to easily view the timeline, events stream, etc., of of existing heap tracing (.svdat) file
+- Add support for Arduino as ESP32 Component
+- Add [support](https://github.com/espressif/vscode-esp-idf-extension#available-commands) for searching ESP-IDF documentation from vscode itself.
+- Fixed some bugs related to [debug adapter](https://github.com/espressif/vscode-esp-idf-extension/pull/134), [gui menuconfig](https://github.com/espressif/vscode-esp-idf-extension/pull/145), [task based commands](https://github.com/espressif/vscode-esp-idf-extension/pull/143), [enhance scss](https://github.com/espressif/vscode-esp-idf-extension/pull/137), etc.
+
```

Comment:
LGTM

I would say that enhance scss PR like UI style improvement for some Webviews.